diff --git a/fieldset/fieldset.d.ts b/fieldset/fieldset.d.ts
index dd1de804c831a751acebb81c1ff67be3a9d4b9db..c63eb6f28f9517bba1a2cb47e7dd38b591b17771 100644
--- a/fieldset/fieldset.d.ts
+++ b/fieldset/fieldset.d.ts
@@ -115,7 +115,7 @@ interface FieldsetToggleEvent {
  * Defines valid properties in Fieldset component. In addition to these, all properties of HTMLFieldSetElement can be used in this component.
  * @group Properties
  */
-export interface FieldsetProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLFieldSetElement>, HTMLFieldSetElement>, 'ref'> {
+export interface FieldsetProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLFieldSetElement>, HTMLFieldSetElement>, 'ref' | 'onToggle'> {
     /**
      * Header text of the fieldset.
      */
diff --git a/inplace/inplace.d.ts b/inplace/inplace.d.ts
index 69edf6612882b775ab5d227c1423b881981ada2a..c5a0918dc4f7902b6afad1593be313790dc0cb1f 100644
--- a/inplace/inplace.d.ts
+++ b/inplace/inplace.d.ts
@@ -89,7 +89,7 @@ interface InplaceToggleEvent {
  * Defines valid properties in Inplace component. In addition to these, all properties of HTMLDivElement can be used in this component.
  * @group Properties
  */
-export interface InplaceProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'ref'> {
+export interface InplaceProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'ref' | 'onToggle'> {
     /**
      * Whether the content is displayed or not. To use in controlled mode you must implement `onToggle` callback at a minimum.
      * @defaultValue false
diff --git a/panel/panel.d.ts b/panel/panel.d.ts
index acaafb239f641c85de193743aad9ec715a1c9686..95f0d3a6b3d770a462d42c4a434aa4295a501f3f 100644
--- a/panel/panel.d.ts
+++ b/panel/panel.d.ts
@@ -184,7 +184,7 @@ export interface PanelState {
  * Defines valid properties in Panel component. In addition to these, all properties of HTMLDivElement can be used in this component.
  * @group Properties
  */
-export interface PanelProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'ref'> {
+export interface PanelProps extends Omit<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'ref' | 'onToggle'> {
     /**
      * Custom header template of the panel.
      */
diff --git a/treeselect/treeselect.d.ts b/treeselect/treeselect.d.ts
index d5cb2fa62a0240d017030bf8b40a07a4f3be6882..53fcc0505c4cfa3e67b3aee8c975157470696c34 100644
--- a/treeselect/treeselect.d.ts
+++ b/treeselect/treeselect.d.ts
@@ -386,7 +386,7 @@ interface TreeSelectNodeDoubleClickEvent extends TreeSelectNodeClickEvent {}
  * Defines valid properties in TreeSelect component. In addition to these, all properties of HTMLDivElement can be used in this component.
  * @group Properties
  */
-export interface TreeSelectProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'onChange' | 'value' | 'ref' | 'pt'> {
+export interface TreeSelectProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'onChange' | 'value' | 'ref' | 'pt' | 'onToggle'> {
     /**
      * DOM element instance where the overlay panel should be mounted. Valid values are any DOM Element and 'self'. The self value is used to render a component where it is located.
      * @defaultValue document.body
diff --git a/treetable/treetable.d.ts b/treetable/treetable.d.ts
index 5139b4beb04faa40065f7f1af9aa92149770a79f..6cad91b116c6e322d593de829df864b440f60613 100644
--- a/treetable/treetable.d.ts
+++ b/treetable/treetable.d.ts
@@ -580,7 +580,7 @@ interface TreeTableColReorderEvent {
  * Defines valid properties in TreeTable component. In addition to these, all properties of HTMLDivElement can be used in this component.
  * @group Properties
  */
-export interface TreeTableProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'size' | 'onContextMenu' | 'onSelect' | 'ref' | 'value'> {
+export interface TreeTableProps extends Omit<React.DetailedHTMLProps<React.InputHTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'size' | 'onContextMenu' | 'onSelect' | 'ref' | 'value' | 'onToggle'> {
     /**
      * Whether to show it even there is only one page.
      * @defaultValue true
